### ISSUE TITLE
Have the pretrained model been used in eval.py？

### ISSUE THREAD

**Issue by mxy519** [Tramac/awesome-semantic-segmentation-pytorch#163]
When I trained the model, the validation result is acceptable, but I run the eval.py, the results is too bad, I want to ask: have the pretrained model been used in eval.py？ I do not read the code about pretrained model.

**Comment by Tramac** [comment]
The pretrained model will be used if [`pretrained=True`](https://github.com/Tramac/awesome-semantic-segmentation-pytorch/blob/c77d34929b43dcc0da13e2452fdc803ddc0b4334/scripts/eval.py#L48).

**Comment by mxy519** [comment]
When I trained the model, the validation result is acceptable, but I run the eval.py, the results is too bad, why is it?

**Comment by Tramac** [comment]
Please confirm whether the pretrained model is loaded normally.

**Comment by mxy519** [comment]
How to confirm it? The pretrained is true

**Comment by mxy519** [comment]
Thanks, this problem has been solved. Can the backbone  be changed to mobilenet?

**Comment by Tramac** [comment]
Yes, you can refer to this [repo](https://github.com/Tramac/Lightweight-Segmentation), the directory structure is similar.

**Comment by mxy519** [comment]
My data is divided into two categories. The predicted mask is black and red. How can I change the predicted mask into black and white
